infrastructure-playbooks/rolling_update.yml

```diff
@@ -115,13 +115,6 @@
       when:
         - not containerized_deployment
 
-  roles:
-    - ceph-defaults
```

Comment:
I think ``ceph-defaults``, ``ceph-common``, ``ceph-docker`` and ``ceph-config`` all need to be here so that the packages actually get upgraded. That package upgrade happens in ``ceph-common`` and ``ceph-docker``. ``ceph-config`` I'm unsure about because that's going to write a new config file, which might be needed depending on if there are config changes needed for the upgrade. Most likely it's harmless though if it's not needed for the upgrade.

Reply:
Everything should be skipped in ceph-common and ceph-docker that doesn't need to change, it's harmless. Those roles will upgrade and restart services as well while handling the logic for different package names, ceph version differences and os differences. I do think we can lose the ceph-$daemon roles though.

Reply:
My concern is more of ceph-osd role, creating devices and partitions stuff. 
I will think about some simpler option to just upgrade the packages in a common way for all node types as required in a rolling update and do the testing and update the PR.
Thanks

Reply:
I understand that concern, but I don't think we need the ceph-osd role here. Removing those daemon specific roles might be the quickest path and should take care of your concern.

Reply:
Thanks for the insights. Am new to ceph/ceph-ansible and thanks for the insights.
I have updated the PR to remove the ceph-osd alone for now.
I have faced issues with raising PR too, then I realized I had to fork and can push only to that forked branch.
I have updated a revert commit, probably rebasing could have been better.
Anyway please have a look at the code change.

Also, is there any way that I can use to run the test cases myself to verify the code before pushing it to the repo?